gui/wxpython/gui_core/ghelp.py

```diff
@@ -330,9 +330,9 @@ def _pageCitation(self):
         """Citation information"""
         try:
             # import only when needed
-            import grass.script as gscript
+            import grass.script as gs
```

Comment:
This function, `_pageCitation(self)`, lazy-loads `grass.script`, but grass.script is already imported at the top of the file. Other parts of the file use that import without lazy-loading. In this function, it is inside a try block, and handles `CalledModuleError`. What do we do here, as it doesn't makes sense.

Reply:
Not sure about the reasoning here, but the lazy import does not seem to be needed.

The CalledModuleError, catches exceptions from g.version. no?

Reply:
Oh, now I see, I'm now able to see where the exception is called and it makes sense. So, I'll remove the lazy import and use the one imported for the file, since it'll work.

Reply:
The PR will need a new approval after having the change committed